docs/GettingStarted.asciidoc

```diff
@@ -264,6 +264,57 @@ Each click point can have an `id`, and if a needle contains multiple click point
 you must pass it to `testapi::assert_and_click` to select which click point
 to use.
 
+=== Configuration
+The different components of openQA read their configuration from the following
+files:
+
+* `openqa.ini`, `openqa.ini.d/*.ini`: These files are the "web UI" config.
+  Services providing the web interface and related services such as the
+  scheduler are configured via these files.
```

Comment:
no problem for now but I found it too generic. And later it says `If these files are not present, defaults are used.` but which is the default if files appears in more than one path